src/components/about/about.module.css

```diff
@@ -0,0 +1,66 @@
+@media (min-width: 390px) {
+
+  .container {
+    margin: 30px 30px 0;
+  }
+
+  .text {
+    width: 330px;
+  }
+
+  .title {
+    font-size: 27px;
+    font-weight: 600;
+    font-family: "Montserrat", sans-serif;
+    margin-top: 30px;
+    margin-bottom: 30px;
+  }
+
+  .description {
+    font-family: "Montserrat", sans-serif;
+  }
+
+  .image {
+    width: 330px;
+    height: 294px;
+    border-radius: 30px;
+  }
+
+  .underline {
+    text-decoration: underline;
+  }
+}
+
+@media (min-width: 1280px) {
```

Comment:
Improve responsiveness between 390px and 1280px. Page should take whole available space of the screen.

Reply:
done

Reply:
<img width="633" alt="Screenshot 2024-04-27 at 13 00 11" src="https://github.com/SerhiiTrokhymets/fe-react-2024/assets/102523692/91656490-07ca-430c-9eae-fbb3b1e0f989">

Reply:
@SerhiiTrokhymets is it clear what do you need to do here?

Reply:
@serhii-tsivinskyi-moc I get the idea, but I'm still figuring it out

Reply:
fixed

Reply:
The image appears oversized below 1280px.

Reply:
@serhii-tsivinskyi-moc 
I'm sorry, I don't understand what to do or what you mean

Reply:
<img width="520" alt="Screenshot 2024-05-04 at 11 28 12" src="https://github.com/SerhiiTrokhymets/fe-react-2024/assets/102523692/a0ac5303-b4e5-4f4e-b478-79e5c8e35458">

@SerhiiTrokhymets an image is huge comparing to a text